.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,62 +1,7 @@
-<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
-</i>
+# Choose a PR Template
 
-### Changelog
-Add a changelog entry for this PR by answering the following questions:
-  1. What's the purpose of the update?
-      - [ ] new service onboarding
-      - [ ] new API version
-      - [ ] update existing version for new feature
-      - [ ] update existing version to fix swagger quality issue in s360
-      - [ ] Other, please clarify
-  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
-  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
-  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
-      - [ ] SDK of .NET (need service team to ensure code readiness)
-      - [ ] SDK of Python
-      - [ ] SDK of Java
-      - [ ] SDK of Js
-      - [ ] SDK of Go
-      - [ ] PowerShell
-      - [ ] CLI
-      - [ ] Terraform
-      - [ ] No refresh required for updates in this PR
+Switch to "Preview" on this description then select one of the choices below.
 
-### Contribution checklist:
-- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
-- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
-- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)
+<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.
```

Comment:
Looks Link is wrong, I tryied it, it reports 404 eror

Reply:
The link is relative so it only works correctly when you start from the expected place, which is within the Pull Request dialog in GitHub. When you request to open a PR in this repo you will be put into a dialog where the URL looks like:

https://github.com/Azure/azure-rest-api-specs/compare/main...<branch>

This is where the PR template will be shown. Clicking the link when on that page will give you a URL that looks like

https://github.com/Azure/azure-rest-api-specs/compare/main...<branch>?expand=1&template=data_plane_template.md

and that link will resolve correctly.  I just tested it again.